doc/en/user/source/community/gsr/functionality.rst

```diff
@@ -0,0 +1,109 @@
+Functionality
+=====================
+
+Currently basic FeatureServer and MapServer functionality work. Each GeoServer workspace is considered
+an ArcGIS "service" for the purposes of the API. ArcGIS URLs look like this in GeoServer:
```

Comment:
ESRI made a standard out of this API over here, we should do our best to reference *GeoServices REST Specification* in these docs:

* http://geoservices.github.io/
* http://www.openwebfoundation.org/

If we do use trademarked words such as ArcGIS and ESRI we should do so carefully and use a footer or something to acknowledge the trademark.

Reply:
fixed

Reply:
Meeting notes, if we wish to use ArcGIS and ESRI we can do so, just note to use "tm" and sphinx footer

Reply:
fixed